### PR TITLE
Introduce stencil-apollo plugin

### PR DESCRIPTION
**Pull request by ardatan**

### CODE REVIEW

**Comment by dotansimha** [comment]
@ardatan still WIP?

**Comment by ardatan** [comment]
@dotansimha It's ready!

File: packages/plugins/typescript-stencil-apollo/.gitignore [hunk 1]
```diff
@@ -0,0 +1,5 @@
+node_modules
```
Comment (dotansimha) [inline packages/plugins/typescript-stencil-apollo/.gitignore]: Please add `example` here

Comment (dotansimha) [reply packages/plugins/typescript-stencil-apollo/.gitignore]: Oh sorry, I meant to add it to `.npmignore` because we don't want to publish it :) @ardatan

Comment (ardatan) [reply packages/plugins/typescript-stencil-apollo/.gitignore]: Ok I see :) But I think you missed this one
https://github.com/dotansimha/graphql-code-generator/blob/stencil-apollo/packages/plugins/typescript-stencil-apollo/.npmignore

**Pull request merged**